golang/wasm/wrapper_js.go

```diff
@@ -172,30 +172,26 @@ func checkKeyStatus(_ js.Value, args []js.Value) any {
 				return
 			}
 
-			switch answerSignalPtr.Rcode {
-			case dns.RcodeSuccess:
-				for _, rr := range answerSignalPtr.Answer {
-					ptrRR, ok := rr.(*dns.PTR)
-					if !ok {
-						err = fmt.Errorf("answer is not a PTR type: %T", rr)
-						reject.Invoke(jsErr(err))
-						return
-					}
-
-					if ptrRR.Ptr == signalKeyRRName {
-						signalPTRExists = true
-						break
-					}
-				}
-
-			case dns.RcodeNameError:
-
```

Comment:
this was a bug. Go doesn't `fallthrough` switch cases without explicitly telling it to.

https://go.dev/wiki/Switch#fall-through